src/ocean/core/UnitTestRunner.d

```diff
@@ -764,8 +764,9 @@ private scope class UnitTestRunner
     {
         // we don't care about freeing anything, is just a few bytes and the program
         // will quite after we are done using these variables
-        char* bin_c = strdup(args[0].ptr);
-        char* prog_c = basename(bin_c);
+        auto prog = args[0].dup;
```

Comment:
Minor note: it predates this patch, but the comment should read `quit`, not `quite` ;-)